documents/data-sharing.md

```diff
@@ -0,0 +1,58 @@
+# Annual survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust project (Core team, Foundation, teams and working groups, contributors).
+* Provide useful data for users and potential users of Rust and providers of Rust tools and services.
+* Provide data in a timely manner.
+* Provide marketing for Rust.
+* Avoid overloading people working directly with the survey data (note that there were over 9000 completed surveys and multiple opportunities for free form responses).
+
+Non-goals:
+
+* We will not aim to make money by selling the data in any form.
+* We will aim to not privilege companies which employ people with access to survey data (raw or processed). However, we acknowledge that controlling widely shared data is impossible.
+
+
+## Blog post
+
+The survey working group will author a blog post for publication on the main Rust blog as soon as possible after the survey is closed. The blog post should acknowledge respondents, and summarise highlights of the survey data. Highlights should be chosen based on being useful or interesting to a general audience (i.e., Rust users and non-users). The blog post should not aim to be a complete summary of the data.
```

Comment:
I might add something to the effect of "The primary intent of the blog post is to communicate interest and usage of Rust, including drilling into interesting categories/use-cases" - maybe there's some other primary goal, but I think something along these lines might be helpful to try and narrow in on in terms of our plan for the blog post.